packages/suite-desktop/scripts/dev.js

```diff
@@ -14,7 +14,8 @@ next.prepare().then(() => {
     const requestHandler = next.getRequestHandler();
     const server = createServer(requestHandler).listen(8000, () => {
         if (!launchElectron) {
-            return;
+            console.error('process.env.LAUNCH_ELECTRON not set to true')
```

Comment:
Not connected with anything in this PR. I was just tinkering with env vars and bumbed into case when launchElectron was false and I ended up waiting forever